### PR TITLE
Update en-update-intro.html

### PR DESCRIPTION
**Pull request by LavredisG**
Think there should be 2 pods instead of 4 at that point.

### CODE REVIEW

**Comment by stmcginnis** [comment]
Actually, it looks like if someone is following through all steps in the tutorial, the last step before this section is to scale back down to 2 pods: https://kubernetes.io/docs/tutorials/kubernetes-basics/scale/scale-intro/

Though with being able to jump in and out of any point in the tutorial, specifying _any_ concrete number here seems slightly risky. But if someone is doing that, I would hope they would not be confused if something isn't exactly like the doc describes.

**Comment by LavredisG** [comment]
What would you suggest in this case? I guess I could close the PR, besides if it were to be fixed, the rest of the languages would need the corresponding change too.

**Comment by LavredisG** [comment]
> Actually, it looks like if someone is following through all steps in the tutorial, the last step before this section is to scale back down to 2 pods: https://kubernetes.io/docs/tutorials/kubernetes-basics/scale/scale-intro/
> 
> Though with being able to jump in and out of any point in the tutorial, specifying _any_ concrete number here seems slightly risky. But if someone is doing that, I would hope they would not be confused if something isn't exactly like the doc describes.

What would you suggest in this case? I guess I could close the PR, besides if it were to be fixed, the rest of the languages would need the corresponding change too.

File: content/en/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 1]
```diff
@@ -175,7 +175,7 @@ <h3>Roll back an update</h3>
                <p>The <code>rollout undo</code> command reverts the deployment to the previous known state (v2 of the image). Updates are versioned and you can revert to any previously known state of a Deployment.</p>
                <p>Use the <code>get pods</code> subcommand to list the Pods again:</p>
                <p><code><b>kubectl get pods</b></code></p>
-               <p>Four Pods are running. To check the image deployed on these Pods, use the <code>describe pods</code> subcommand:</p>
+               <p>Two Pods are running. To check the image deployed on these Pods, use the <code>describe pods</code> subcommand:</p>
```
Comment (divya-mohan0209) [inline content/en/docs/tutorials/kubernetes-basics/update/update-intro.html:178]:
```suggestion
               <p>To check the image deployed on the running Pods, use the <code>describe pods</code> subcommand:</p>
```

File: content/en/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 2]
```diff
@@ -175,7 +175,7 @@ <h3>Roll back an update</h3>
                <p>The <code>rollout undo</code> command reverts the deployment to the previous known state (v2 of the image). Updates are versioned and you can revert to any previously known state of a Deployment.</p>
                <p>Use the <code>get pods</code> subcommand to list the Pods again:</p>
                <p><code><b>kubectl get pods</b></code></p>
-               <p>Four Pods are running. To check the image deployed on these Pods, use the <code>describe pods</code> subcommand:</p>
+               <p>To check the image deployed on the running Pods, use the <code>describe pods</code> subcommand:</p>
```
Comment (network-charles) [inline content/en/docs/tutorials/kubernetes-basics/update/update-intro.html:178]: This looks good, I’ll suggest you squash your commits so it's ready to be merged.

Comment (LavredisG) [reply content/en/docs/tutorials/kubernetes-basics/update/update-intro.html:178]: done

Comment (network-charles) [reply content/en/docs/tutorials/kubernetes-basics/update/update-intro.html:178]: cc @tengqm

**Comment by network-charles** [comment]
Two LGTM labels are required for it to be merged.

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!

/lgtm

**Pull request merged**